pontoon/pretranslation/pretranslate.py

```diff
@@ -18,6 +18,7 @@
     get_reconstructed_message,
 )
 
+UNTRANSLATABLE_KEY = "AIzaSyDX3R5Y1kxh_8lJ4OAO"
```

Comment:
Why don't we simply cut the `key` off and add it back when build a pretranslated message?

Reply:
No extra code to cut off and reinsert the key (replace works fine). I will need to handle cases where there might be spaces around the `=` separator and then add them back (probably). 

A variation will work well for attributes as well. While removing attributes might be a hassle

Reply:
We really ought to extract each flat pattern, and have the ability to re-integrate it in its context. Consider for instance the following real-world message:

```fluent
detail-show-preferences =
    .label =
        { PLATFORM() ->
            [windows] Options
           *[other] Preferences
        }
    .tooltiptext =
        { PLATFORM() ->
            [windows] Change this add-on’s options
           *[other] Change this add-on’s preferences
        }
```

Here, we should be able to find and separate the four patterns in this message, translate each separately, and then feed them back into the message structure at the appropriate slots.

Reply:
The ID getting translated is [the most pressing issue](https://github.com/mozilla/pontoon/issues/2691#issuecomment-1397546569), and #2690 is only about that. We'd like to fix that ASAP.

We've talked about "protecting" other parts of the syntax, and my suggestion was to only focus on the message ID.

Reply:
Let's continue with `UNTRANSLATABLE_KEY` for now.